src/components/DevMenu.svelte

```diff
@@ -0,0 +1,15 @@
+<script>
+  import { createChatroom, db } from "../utils/api";
```

Comment:
as previous comment from the routing path

Reply:
See above